rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1547,7 +1550,7 @@ void game_list_frame::ShowCustomConfigIcon(QTableWidgetItem* item)
 	else if (game->hasCustomPadConfig)
 		m_gameList->item(item->row(), gui::column_name)->setIcon(QIcon(":/Icons/controllers.png"));
 	else
-		m_gameList->setItem(item->row(), gui::column_name, new custom_table_widget_item(game->info.name));
```

Comment:
I think you can just use setIcon(QIcon()) here.
Then you can scope the custom_title code to the grid path only.
I don't remember why I used setItem. I guess it doesn'T really matter.

In general I think you have to fix a typo in line 1067 to even be able to test this with custom pad configs.
Replace GetCustomConfigPath with GetCustomInputConfigDir